.circleci/config.yml

```diff
@@ -107,6 +107,19 @@ jobs:
       - restore_cache:
           keys:
             - habitat-sim-{{ checksum "./hsim_sha" }}
+      - restore_cache:
+          keys:
+            - ccache-{{ arch }}-{{ .Branch }}-{{ epoch }}
```

Comment:
Won't this not work as `{{ epoch }}` is the current time and the cache-key will forever be different?  Since this is the API, probably `ccache-{{ arch }}-{{ checksum "./hsim_sha" }}` makes sense as the key.

Reply:
Actually, it should just be `ccache-{{ arch }}` as the install of the simulator is already cached.